### ISSUE TITLE
GMB listing_udhira pote

### ISSUE THREAD

**Issue by udhirapote146** [LetsUpgrade/Digital-Marketing-Essentials#22]
[GMB listing_udhira pote.docx](https://github.com/LetsUpgrade/Digital-Marketing-Essentials/files/5113949/GMB.listing_udhira.pote.docx)

**Comment by udhirapote146** [comment]
What happened??

**Comment by anvsm** [comment]
You need to Submit the assignment in the assignment submission form. Not here. So , only i closed it.

**Comment by udhirapote146** [comment]
In form we have to give GitHub link is it right?? Ultimately you received
my assignment or not?

**Comment by anvsm** [comment]
Yes, you need to give your own github account link in the Submission form.

**Comment by udhirapote146** [comment]
i have done that will you let me know you received right information or not?
